DuckDuckGo/MainViewController.swift

```diff
@@ -1002,11 +1002,7 @@ extension MainViewController: BookmarksDelegate {
     }
     
     func bookmarksUpdated() {
-        if bookmarkStore.favorites.isEmpty {
-            homePageChanged()
-        } else {
-            homeController?.refresh()
-        }
+        homePageChanged()
```

Comment:
This has been working for favorites -> no favorites state transition, but not the other way around.